lib/fluent/plugin/in_tail.rb

```diff
@@ -911,16 +1000,35 @@ def opened?
         def handle_notify
           with_io do |io|
             begin
+              number_bytes_read = 0
+              start_reading = Fluent::EventTime.now
```

Comment:
Probably it should be `Fluent::Clock.now` instead of `Fluent::EventTime.now`.
Otherwise it may cause unexpected high load when system time is changed.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3185/commits/2a994ab074cee483723e3489dffc2816ec753ce5.